catalog/src/androidMain/AndroidManifest.xml

```diff
@@ -6,6 +6,7 @@
     <application
         android:allowBackup="false"
         android:supportsRtl="true"
+        android:usesCleartextTraffic="true"
```

Comment:
nit - should we use a network policy xml instead of doing this just to enable only for debug?